inc/search.class.php

```diff
@@ -1135,8 +1265,10 @@ static function constructCriteriaSQL($criteria = [], $data = [], $searchopt = []
     * @param  array  &$data                TODO: should be a class property (output parameter)
     *
     * @return void
+    *
+    * @since 10.0.0 Method is no longer static
     */
-   static function constructAdditionalSqlForMetacriteria($criteria = [],
+   public function constructAdditionalSqlForMetacriteria($criteria = [],
```

Comment:
1. Should be mentionned in changelog.

2. This method is only privately used, may be changed to private.